packages/vaadin-lumo-styles/src/props/icons.css

```diff
@@ -0,0 +1,54 @@
+@font-face {
+  font-family: 'lumo-icons';
```

Comment:
Could we also update `gulpfile.cjs` in the Lumo package to also generate this file when running `yarn icons`?

Reply:
Done.